grpc.sh

```diff
@@ -22,6 +22,13 @@ git checkout "$GIT_TAG"
 git submodule update --init
 popd
 
+case $ARCHITECTURE in
+  osx*) 
+    [[ ! $OPENSSL_ROOT ]] && BOOST_ROOT=$(brew --prefix openssl)
```

Comment:
maybe that's because of the heat, but I don't quite understand the logic of these two new lines here (boost <-> openssl ; openssl <-> Protobuf)

Reply:
It's 40 in my office...